### ISSUE TITLE
Add Annotation statistics

### ISSUE THREAD

**Issue by bbernhard** [bbernhard/imagemonkey-core#50]
I think it could be useful to display a small graphical representation (pie chart, bar chart..) that visualizes how many people have validated the annotation (num_of_yes vs num_of_no). 

We don't necessarily have to show that by default (if that distracts too much) and maybe only blend it in if the user clicks a button, but I think it could be useful and motivating for other users. 

I am thinking about something like this: 

![annotation_valid_statistics](https://user-images.githubusercontent.com/5710631/33151946-7e4b6f42-cfda-11e7-9f85-5584fcdbab95.png)

**Comment by dobkeratops** [comment]
possibly show it afterward so it doesn't bias your judgement

**Comment by bbernhard** [comment]
You are right, showing the data upfront could bias your judgement.

My only concern with the "showing afterwards" approach is, that this could maybe "break" the validation flow a little bit. Because then you would see the "95% agree" after every validation...even if you might not be interested in it. We could probably jump to the next annotation automatically after a few seconds, but you would "lose" a few seconds after every validation. On the other hand, feedback could also be useful and motivating... 

Another try (It was before I read your suggestion...but now you got me thinking ;-)):

![feedback](https://user-images.githubusercontent.com/5710631/33189628-0939fb4e-d0a5-11e7-9807-de15d916be1a.gif)

**Comment by dobkeratops** [comment]
> that this could maybe "break" the validation flow a little bit.

agree.. if it had to show another screen between each validation, that would be obtrusive.
how about showing the previous validation&'other people say..' as a thumbnail to one side, whilst showing the current.
